src/SDKs/CognitiveServices/dataPlane/Search/BingCustomImageSearch/BingCustomImageSearch.Tests/Microsoft.Azure.CognitiveServices.Search.CustomImageSearch.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <Description>Microsoft.Azure.CognitiveServices.Search.CustomImageSearch.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.Search.CustomImageSearch.Tests</AssemblyName>
+    <VersionPrefix>1.1.0-preview</VersionPrefix>
```

Comment:
Set the test project version to `1.0.0` this is never published

Reply:
Updated